python/wave_saver.py

```diff
@@ -31,7 +31,12 @@ def save(self, filename):
             params = (self._nchannels, self._sampwidth, self._framerate, len(self.buffer), 'NONE', 'not compressed')
             w.setparams(params)
             w.writeframes(self.buffer)
-    
+
+    def saveOggVorbis(self, filename):
```

Comment:
This class should be renamed to `AudioSaver`. If you move passing encoding type to AudioSaver constructor, only one `save` method could be exposed;

Reply:
Done.